apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V96__recreate_pre_adj_child_tables.sql

```diff
@@ -0,0 +1,198 @@
+-- Rebuild the schema to match new RDA specifications and truncate tables to end soft launch
+
+drop table rda.fiss_audit_trails;
+drop table rda.fiss_diagnosis_codes;
+drop table rda.fiss_payers;
+drop table rda.fiss_proc_codes;
+drop table rda.mcs_adjustments;
+drop table rda.mcs_audits;
+drop table rda.mcs_details;
+drop table rda.mcs_diagnosis_codes;
+drop table rda.mcs_locations;
+truncate table rda.fiss_claims;
+truncate table rda.mcs_claims;
+truncate table rda.message_errors;
+truncate table rda.rda_api_progress;
+-- Not truncating the metadata or MBI tables
+alter table rda.claim_message_meta_data ${logic.alter-rename-column} received_date ${logic.rename-to} last_updated;
+alter index rda.claim_message_meta_data_received_date_idx rename to claim_message_meta_data_last_updated_idx;
+
+/*
+ * fiss_audit_trails
+ */
+CREATE TABLE rda.fiss_audit_trails (
+   dcn varchar(23) NOT NULL,
+   badt_status varchar(1),
+   badt_loc varchar(5),
+   badt_oper_id varchar(9),
+   badt_reas varchar(5),
+   badt_curr_date date,
+   rda_position smallint NOT NULL,
```

Comment:
small thing, but would you mind ordering the dcn and rda_position columns next to each other